hibernate-core/src/main/java/org/hibernate/bytecode/enhance/internal/AttributeTypeDescriptor.java

```diff
@@ -51,7 +49,7 @@ else if ( currentValue.getType().getName().startsWith( "java.lang" )
 					|| currentValue.getType().getName().startsWith( "java.sql.Date" )
 					|| currentValue.getType().getName().startsWith( "java.util.Date" )
 					|| currentValue.getType().getName().startsWith( "java.util.Calendar" ) ) {
```

Comment:
Just noticed these.  I think we should add the Java 8 date/time types here as well